### ISSUE TITLE
Release for FIP-0090/0092

### ISSUE THREAD

**Issue by BigLep** [filecoin-project/filecoin-ffi#457]
Tracking issue for doing a filecoin-ffi release for FIP-0090/0092 in support of nv23.

This will follow this process: https://github.com/filecoin-project/filecoin-ffi?tab=readme-ov-file#development-guidelines-wip

```[tasklist]
### Depends on
- [ ] https://github.com/filecoin-project/filecoin-ffi/pull/453
- [ ] https://github.com/filecoin-project/filecoin-ffi/pull/456
- [ ] https://github.com/filecoin-project/filecoin-ffi/pull/458
- [ ] https://github.com/filecoin-project/filecoin-ffi/pull/459
```

**Comment by rjan90** [comment]
I have published a [filecoin-ffi dev-release (v1.28.0-dev2) ](https://github.com/filecoin-project/filecoin-ffi/releases/tag/v1.28.0-dev2) based on the latest commit in PR #459. The last commit in that PR patches rust-fil-proofs to #1758, unblocking testing in Butterfly until a proper rust-fil-proofs release is available.

**Comment by BigLep** [comment]
It looks like we have the rust-fil-proofs release per https://github.com/filecoin-project/rust-fil-proofs/pull/1758 thus should be able to close this one out.

**Comment by BigLep** [comment]
Marking as done/closed since @rjan90 had marked it as such in https://docs.google.com/document/d/1MOHg6HoFzqKP3EZ_ptBL9sOht0AP8cnYoWFkmRH5MZo/edit

**Comment by BigLep** [comment]
2024-06-24 conversation: reopened because we want to have a filecoin-ffi dev-release (v1.28.0-rc1) with  the release that corresponds with https://github.com/filecoin-project/rust-fil-proofs/pull/1758

**Comment by rjan90** [comment]
> 2024-06-24 conversation: reopened because we want to have a filecoin-ffi dev-release (v1.28.0-rc1) with the release that corresponds with filecoin-project/rust-fil-proofs#1758

Hey! I should have updated this issue ticket before I went OOO. But the filecoin-ffi release ([v1.28.0-rc1](https://github.com/filecoin-project/filecoin-ffi/releases/tag/v1.28.0-rc1) includes an update to rust-fil-proofs 18.1 in this commit https://github.com/filecoin-project/filecoin-ffi/commit/3bca86b08fed052fdd98e4845499e5d6b54f56e5 which is a patch release that includes filecoin-project/rust-fil-proofs#1758. 

Therefore there should not be a need of cutting another filecoin-ffi release, unless there has been other breaking changes since I went OOO.
